### ISSUE TITLE
Error in:  Using search query URL

### ISSUE THREAD

**Issue by manyu90** [kundajelab/ENCODE_downloader#4]
I tried running the search query URL and got the following error:

I ran 
```
python ENCODE_downloader.py "https://www.encodeproject.org/search/?type=Experiment&assay_title=ATAC-seq&replicates.library.biosample.life_stage=postnatal&status=released" --pipeline-encode-lab /labs/anshul-kundaje/ --pipeline-encode-award /awards/U41HG007000/ --pipeline-encode-alias-prefix anshul-kundaje --pipeline-encode-award-rfa ENCODE3 --ignore-unpublished
```
and got : 

```
File "ENCODE_downloader.py", line 137
    if debug: print '{}: {}'.format(help_str, str(json))
                           ^
SyntaxError: invalid syntax
```

**Comment by leepc12** [comment]
Fixed in the commit cd359127b735ad28ad36089c65450fca9a0b7967. Now it works with python3 too.

**Comment by manyu90** [comment]
Thanks for fixing this.
